src/ViewerComponents/ImageViewer.svelte

```diff
@@ -0,0 +1,120 @@
+<script>
```

Comment:
To use these viewers to display search results, these viewers should be able to be called with the resource to display already loaded (by passing the json/the necessary information)

Reply:
Will happen later

Reply:
I will implement this later